__tests__/app.test.js

```diff
@@ -0,0 +1,268 @@
+        expect(review).toHaveProperty("votes", expect.any(Number));
+        expect(review).toHaveProperty("category", expect.any(String));
+        expect(review).toHaveProperty("review_body", expect.any(String));
+        expect(review).toHaveProperty("owner", expect.any(String));
+        expect(review).toHaveProperty("created_at", expect.any(String));
+      });
+  });
+  it("if the request is for an id which doesn't exist, responds with status 404", () => {
+    return request(app)
+      .get("/api/reviews/100")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("404: No user found for user_id: 100");
+      });
+  });
+  it("if the request is ill-formed, responds with status 400", () => {
+    return request(app)
+      .get("/api/reviews/cats")
+      .expect(400)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("400: ill-formed request");
+      });
+  });
+});
+
+describe("GET /api/reviews", () => {
+  it("should respond with a reviews array of review objects, each of which should have corresponding properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        for (let review of reviews) {
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("category", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("created_at", expect.any(String));
+          expect(review).toHaveProperty("votes", expect.any(Number));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          if (review.id === 2 || review.id === 3) {
+            expect(review.comment_count).toBe(3);
+          } else {
+            let comment_count = Number(review.comment_count);
+            expect(typeof comment_count).toBe("number");
+          }
+        }
+      });
+  });
+  it("the array should be in descending order", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        for (let i = 1; i < reviews.length; i++) {
+          let result = reviews[i - 1].created_at >= reviews[i].created_at;
+          expect(result).toBe(true);
+        }
+      });
+  });
+});
+
+describe("PATCH /api/reviews/:review_id", () => {
+  it("responds with the updated review(checking for review with id =1 and increases votes by 1)", () => {
+    const numberOfVotesToAdd = 1;
+    return request(app)
+      .patch("/api/reviews/1")
+      .send({ inc_votes: numberOfVotesToAdd })
+      .expect(200)
+      .then(({ body }) => {
+        const { updatedReview } = body;
```

Comment:
I would keep this on a key of just review, rather than updatedReview. It helps with consistency when clients are accessing different endpoints of your BE